### DISCUSSION TITLE
Linux colobot builds is broken?

### DISCUSSION THREAD

**Discussion by X64E** [colobot/colobot#1482] [Q&A]
Checked on Ubuntu 20.04.2 LTS

https://nightly.link/colobot/colobot/workflows/build/dev/linux-debug-AppImage.zip
with sources:
https://nightly.link/colobot/colobot-data/workflows/build/dev/data.zip
Build does not start. If run it from terminal result in log.txt is:
          
          [INFO]: Colobot: Gold Edition 0.2.0+alpha-git-dev~r9fd9357 starting
          [INFO]: Creating CApplication
          [ERROR]: Data directory '/tmp/.mount_coloboy14cbv/usr/bin/data' doesn't exist or is not a directory
          [INFO]: Didn't run main loop. Exiting with code 1

![изображение](https://user-images.githubusercontent.com/95232774/144803051-0a6de98e-22cf-467e-a695-805291581eb9.png)

Whats wrong?

Is possible to run game under Linux?

**Comment by MrSimbax** [comment]
AppImage produced by the CI does not contain the `data` folder as `data` is built separately. The built `data` folder has to be downloaded separately (for example, from the CI build of [`colobot-data`](https://github.com/colobot/colobot-data) at matching commit). Then run the AppImage with the `--data-dir` command-line argument specifying the path to the `data` folder or specify it with the `COLOBOT_DATA_DIR` environment variable. You can also use the latest release's `data` but it is not guaranteed to work as the nightly binary might require changes in `data`.

Note that the binaries produced by CI are intended to be used for testing purposes only.
